py3status/modules/sysdata.py

```diff
@@ -163,8 +212,10 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     format = "[\?color=cpu CPU: {cpu_usage}%], " \
-             "[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]"
+             "[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]" \
+             "[\?color=mem Swap: {swap_used}/{swap_total} GB ({swap_used_percent}%)]"
```

Comment:
I'd use `[\?not_zero=mem Swap: {swap_used}/{swap_total} GB ({swap_used_percent}%)]` instead.
Idk how to combine it with \?color= ... No available formatter docs.